addons/website_slides/controllers/main.py

```diff
@@ -336,30 +342,39 @@ def slides_channel_home(self, **post):
             'top3_users': self._get_top3_users(),
             'challenges': challenges,
             'challenges_done': challenges_done,
-            'search_tags': request.env['slide.channel.tag']
+            'search_tags': request.env['slide.channel.tag'],
+            'slide_url': QueryURL('/slides/all', ['tag']),
+            'tags_list': self.tags_list,
         })
 
         return request.render('website_slides.courses_home', values)
 
-    @http.route('/slides/all', type='http', auth="public", website=True, sitemap=True)
-    def slides_channel_all(self, slide_type=None, my=False, **post):
+    @http.route(['/slides/all', '/slides/all/tag/<string:tags>'], type='http', auth="public", website=True, sitemap=True)
+    def slides_channel_all(self, slide_type=None, tags=None, my=False, **post):
         """ Home page displaying a list of courses displayed according to some
         criterion and search terms.
 
           :param string slide_type: if provided, filter the course to contain at
            least one slide of type 'slide_type'. Used notably to display courses
            with certifications;
+          :param string tag: if provided, filter the slide.channels having
+            the tag(s)
           :param bool my: if provided, filter the slide.channels for which the
            current user is a member of
           :param dict post: post parameters, including
 
            * ``search``: filter on course description / name;
-           * ``channel_tag_id``: filter on courses containing this tag;
-           * ``channel_tag_group_id_<id>``: filter on courses containing this tag
-             in the tag group given by <id> (used in navigation based on tag group);
         """
+
+        if tags and request.httprequest.method == 'GET':
```

Comment:
It's more related to website SEO than website_slides. We avoid multi tags in GET request to avoid stupid bot indexation. See this reference commit - https://github.com/odoo/odoo/commit/0f2cada32319b3910d6ace6d412cfa94a646c9c7#diff-43da97601625bf08baa784f17cd695f4R160 (which is already there in the commit message for explanation). @JKE-be would be explain it better than me. Also, I've added few comments about post.get('search') part.